app/utils/tests/sagaInjectors.test.js

```diff
@@ -103,7 +101,7 @@ describe('injectors', () => {
       process.env.NODE_ENV = originalNodeEnv;
     });
 
-    it('should not remove daemon saga\'s descriptor in production', () => {
+    it("should not remove daemon saga's descriptor in production", () => {
```

Comment:
This looks a little strange

Reply:
so prettier's default formatting behavior (both with and without --no-semi option) is to prefer double quotes instead of escape characters... it converts string literals to use double quotes so there's no need to escape the tick...
```js
it('should not remove daemon saga\'s descriptor in production', () => {})
```
becomes
```js
it("should not remove daemon saga's descriptor in production", () => {});
```
and 
```js
it('should not remove daemon saga descriptor in production', () => {});
```
stays the same since no escape char
```

afaik there's no way to change this

Reply:
Yes there is no way to change this. I the prettier [docs](https://prettier.io/docs/en/options.html#quotes) it gives the notes:
>If the number of quotes outweighs the other quote, the quote which is less used will be used to format the string - Example: "I'm double quoted" results in "I'm double quoted" and "This \"example\" is single quoted" results in 'This "example" is single quoted'.

@justingreenberg do you thinks this is a risk?

Reply:
BTW, I also found:
`it("should remove non daemon saga\"'s descriptor in production", () => {});`
becomes
`it('should remove non daemon saga"\'s descriptor in production', () => {});`

Reply:
@blling totally cool with me—small price to pay imo :)